python/servo/build_commands.py

```diff
@@ -175,21 +175,22 @@ def build(self, target=None, release=False, dev=False, jobs=None,
             print("Please specify either --dev or --release.")
             sys.exit(1)
 
-        targets = []
+        if target and android:
+            print("Please specify either --target or --android.")
+            sys.exit(1)
+
         if release:
             opts += ["--release"]
-        if target:
-            opts += ["--target", target]
-            targets.append(target)
         if jobs is not None:
             opts += ["-j", jobs]
         if verbose:
             opts += ["-v"]
         if android:
-            opts += ["--target", self.config["android"]["target"]]
-            targets.append("arm-linux-androideabi")
+            target = self.config["android"]["target"]
 
-        self.ensure_bootstrapped(targets=targets)
+        if target:
+            opts += ["--target", target]
+            self.ensure_bootstrapped(target=target)
```

Comment:
OK, let's put the comment in the right place this time: `ensure_bootstrapped` also sets up the host bits (cargo, etc.), so it shouldn't be indented. The naming of the target kwarg seems a bit confusing - maybe consider renaming it to extra_target? Bikeshedding welcome.